apps/site/lib/site_web/templates/page/_whats_happening.html.eex

```diff
@@ -5,7 +5,7 @@
       <h2 class="sr-only">Featured MBTA Updates and Projects</h2>
     <% end %>
     <div class="container page-section">
-      <%= unless @promoted do %>
+      <%= unless @promoted == false do %>
```

Comment:
this is kind of confusing and I think is the opposite of the intended effect. There's 2 versions of this template being rendered, one for promoted and one for not promoted. This header should be present for the not promoted so `if @promoted == false` or `unless @promoted` like it already was. The `<%= if @whats_happening_items %>` at the top should cover the rendering of the section entirely if there is no content. 

From what I can tell the ticket (and possibly dev green) are outdated compared to the current state of the code.

Reply:
I agree that this is confusing; I'd expect some logic checking the `@whats_happening_items` list being rendered below.